### DISCUSSION TITLE
Feature Backport runWithContext

### DISCUSSION THREAD

**Discussion by ColinRosati** [nuxt/bridge#757] [General]
**Describe the Feature**
Original feat request [issue](https://github.com/nuxt/nuxt/issues/14805)

**Why** 
- To keep nuxt bridge up to date with nuxt 3 we should back port this runWithContext helper on nuxt app

**What** 
- The proposed changes would be to backport the feature to nuxt-bridge: Update nuxtApp decleration & type definition

Related:
[nuxt 3 runWithContext](https://github.com/nuxt/nuxt/pull/20608)

**Comment by ColinRosati** [comment]
I have a proposed solution I would like to open an PR with after initial triage :)

**Reply by AndreyYolkin** [reply]
I will (hopefully) pass pre-presentation my master's dissertation tomorrow, after that I can help to backport this feature. Mention me if you want to collaborate)
